app/assets/javascripts/smart-answers.js

```diff
@@ -1,7 +1,13 @@
+//= require start-button-ab-test-july-2016
+
 function browserSupportsHtml5HistoryApi() {
   return !! (history && history.replaceState && history.pushState);
 }
 
+function isExpectedPath(slug) { // Used mostly during A/B testing
```

Comment:
It is not clear to me what this method is doing, because the argument is not a path, it is a smart-answer name. It might make be better to rename it to something similar to:

```ruby
SmartAnswer.isStartPage(‘overseas-passports’)
``` 

If there are any particular reasons for doing `split("/").join("")` it would be good to explain it in the commit notes.

I am not sure either about having this function isolated. I would prefer so see it enclosed within an object / module. I will facilitate maintenance and testing.